### DISCUSSION TITLE
Custom domain showing on the web UI but I don't have access to it whenever I try to alias a deployment

### DISCUSSION THREAD

**Discussion by KevinBacas** [vercel/community#7357] [Help]
### Summary

I'm trying to alias a deployment to "preprod.augora.fr" but the CLI keeps telling me that we don't have acces to augora.fr domain.

I also can't see any domains attached to this account using the CLI, but on the web page I can see Augora.fr.

What am I supposed to do to add this domain and use it properly ? I recently  changed the nameservers to vercel instead of Cloudflare but I still have the same issue.

Can you help me figure out how to get access to augora.fr with this account ?

### Steps to Reproduce

Here is basically what I did :

![image](https://github.com/vercel/community/assets/1434393/2db7ad89-97bb-4017-8d2f-472cf937a718)

I can see that no domains are accessible from this account (even if the web ui is showing augora.fr) and when I try to add it, it says that this domain is assigned to another project :
![image](https://github.com/vercel/community/assets/1434393/30e6fcf6-facc-49ed-90a0-574205a3eb5c)

**Comment by pawlean** [comment]
Hi @KevinBacas,

>  I recently changed the nameservers to vercel instead of Cloudflare but I still have the same issue.

How long ago was this? Wondering if it's a DNS issue.

**Reply by KevinBacas** [reply]
Hello, I changed it around 4 hours ago.

But it was not working before this as well, changing the nameservers was one attempt to solve this on my end :)

[a comment by an automated account is not shown]
